docusaurus/docs/dev-docs/migration/v4-to-v5/breaking-changes/upgrade-to-apollov4.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Apollo Server v3 upgraded to Apollo Server v4
+description: The upgrade from Apollo Server v3 to v4 and graphql ^15 to ^16.
+sidebar_label: Upgrade to Apollo Server v4
+displayed_sidebar: devDocsMigrationV5Sidebar
+tags:
+ - breaking changes
+ - Content API
+ - GraphQL
+ - GraphQL API
+---
+
+import FeedbackCallout from '/docs/snippets/backend-customization-feedback-cta.md'
+import Intro from '/docs/snippets/breaking-change-page-intro.md'
+import MigrationIntro from '/docs/snippets/breaking-change-page-migration-intro.md'
+
+# Apollo Server v3 upgraded to Apollo Server v4
+
+Strapi 5 has some changes for user code that relied on specific versions.
+<Intro />
+
+## Breaking change description
+
+<SideBySideContainer>
+
+<SideBySideColumn>
+
+**In Strapi v4**
+
+Apollo Server v3 for the GraphQL server and graphql ^15 for the GraphQL module.
+
+</SideBySideColumn>
+
+<SideBySideColumn>
+
+**In Strapi 5**
+
+Apollo Server v4 for the GraphQL server and graphql ^16 for the GraphQL module.
+
+</SideBySideColumn>
+
+</SideBySideContainer>
+
+:::callout
```

Comment:
I'd suggest you add information from these 2 callouts to the notes section instead. You don't need to use callouts, just use the bullet points in notes.